R/summarise_scores.R

```diff
@@ -14,6 +14,10 @@
 #' indicate indicate a grouping of forecasts (for example there may be one
 #' forecast per day, location and model). Adding additional, unrelated, columns
 #' may alter results in an unpredictable way.
+#' @param across character vector with column names from the vector of variables
```

Comment:
We might want to add something like "scores will be summarised across the columns specified (meaning that the corresponding columns are dropped afterwards)". 

Don't know.

Reply:
Maybe it's clear to users. Just thought it could be a bit more verbose what happens.

Reply:
Made it more verbose in 0891dfd13828adeca3501194e275c7f7af9c252f